app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -1,11 +1,28 @@
 import * as yup from "yup";
 
+import { convertDateToStdFormat } from "./utils";
+
+const DUMMY_CONTACT_DATA = {
+  name: "Ronald Reagan",
+  email: "ronald@gmail.com",
+  createdAt: convertDateToStdFormat(new Date()),
+};
+
+export const ROW_DATA = Array(100).fill(DUMMY_CONTACT_DATA);
+
 export const ROLES = [
-  "UX designer",
-  "Tester",
-  "Analyst",
-  "Back-end developer",
-  "Front-end developer",
+  {
+    label: "UX designer",
+    value: "uxDesigner",
+  },
+  {
+    label: "Tester",
+    value: "tester",
+  },
+  {
+    label: "Analyst",
+    value: "analyst",
+  },
```

Comment:
Create a `buildSelectOption` global(should be kept at `src/utils/index.js`) utility function that can provide `label` and camelCased `value`.
```suggestion
export const ROLES = [
  "UX designer",
  "Tester",
  "Analyst",
  "Back-end developer",
  "Front-end developer",
].map(buildSelectOption);
```

Reply:
I have few doubts regarding this , what exactly does the buildSelectOption function do? Does it just convert label to camel case and return an Object like this for value in array :-  { 
                                                                                label : "String Normal",
                                                                                value: "stringNormal",
                                                                           }

Reply:
@eshaanks Yes. Exactly.

Reply:
I have already built the former functionality , so can i strick to it?

Reply:
@eshaanks You can actually change the logic to https://github.com/eshaanks/neeto-ui-challenge/pull/19#discussion_r973035094 since that is a better implementation.

Reply:
I have made changes , will be reflected in the next PR.